src/Employer/Employer.Web/Views/PartialNames.cs

```diff
@@ -4,6 +4,8 @@ public static class PartialNames
     {
         public const string DisabilityConfident = "_DisabilityConfidentPartial";
         public const string GoogleAnalyticsSnippet = "_GoogleAnalyticsPartial";
+        public const string GoogleTagManagerHeadPartial = "_GoogleTagManagerHeadPartial";
```

Comment:
can delete this `public const string GoogleAnalyticsSnippet = "_GoogleAnalyticsPartial";`

Reply:
Will remove.